common.sh

```diff
@@ -73,10 +73,6 @@ export SSH_PUB_KEY="${SSH_PUB_KEY:-$(cat $HOME/.ssh/id_rsa.pub)}"
 
 # mirror images for installation in restricted network
 export MIRROR_IMAGES=${MIRROR_IMAGES:-}
-# For IPv6 (default case) mirror images are used since quay doesn't support IPv6
```

Comment:
I added this because I change the condition on [L488](https://github.com/openshift-metal3/dev-scripts/pull/1694/files#diff-a045819fa24dfbeb74522ddaf6eac77b5a4d39762752db45251e6101cd5c987dL488).

I wonder if the IPv6 jobs failing is related to this? If it is, we may end up with better result if we remove the first condition `-z "${IP_STACK:-}"` only here?

Reply:
Given that in `common.sh` there was already a point where `MIRROR_IMAGES` was set on [L223](https://github.com/openshift-metal3/dev-scripts/pull/1694/files#diff-a045819fa24dfbeb74522ddaf6eac77b5a4d39762752db45251e6101cd5c987dL223-L225) - and that's usually a dev-only feature when making a [custom release](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/docs/custom-release.md) not used in CI -, my personal preference would be to limit the number of places where a config var is set to avoid increasing the complexity. 
IIRC the block on L488 it's not actively used in CI (cc @bfournie for confirmation), so I don't think it's an issue right now (and I feel it could be moved somewhere else eventually)

Reply:
> I wonder if the IPv6 jobs failing is related to this?

I suspect it's something else  - at least for the agent jobs, since locally it's working (maybe @elfosardo or @derekhiggins could comment on the metal jobs)

Reply:
It was not, in my local tests I wasn't using the `AGENT_USE_ZTP_MANIFESTS` config var (as it happens in the agent SNO CI job). So, I restored the initial checks to allow defining `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`, and now it should be ok (unfortunately the CI is broken at the moment...)

Reply:
@andfasano correct, the `oc-mirror` case is not currently used in CI, when we looked into using it did not support nightly builds (https://issues.redhat.com/browse/AGENT-368). If that has changed we should make the default for mirroring since its a more common use case.